types/emartech__cls-adapter/index.d.ts

```diff
@@ -0,0 +1,65 @@
+// Type definitions for @emartech/cls-adapter 1.3
+// Project: https://github.com/emartech/cls-adapter
+// Definitions by: Lex Webb <https://github.com/lexwebb>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+
+import * as Koa from "koa";
+import * as Express from 'express';
+import * as CLS from "cls-hooked";
+
+declare class ClsAdapter {
+    private _namespace: CLS.Namespace;
```

Comment:
```suggestion
    private _namespace: CLS.Namespace;
```
It really cannot be accessed outside of the class itself, IMO not required here at all, unless for specific reason

Reply:
This is a good question. The linter doesn't like having a class with no instanced feilds on, and wanted me to convert it to an object. I thought however that since the original JS implementation is a class i should keep it that way, and adding this field made the linter happy. 

It was that or disable the linter rule, which the guidelines suggest against